echopype/convert/api.py

```diff
@@ -342,11 +345,21 @@ def open_raw(
     convert_params : dict
         parameters (metadata) that may not exist in the raw file
         and need to be added to the converted file
-    storage_options : dict
+    storage_options : dict, optional
         options for cloud storage
     use_swap: bool
+        **DEPRECATED: This flag is ignored**
         If True, variables with a large memory footprint will be
         written to a temporary zarr store at ``~/.echopype/temp_output/parsed2zarr_temp_files``
+    destination_path: str
+        The path to a swap directory in a case of a large memory footprint.
+        This path can be a remote path like s3://bucket/swap_dir.
+        By default, it will create a temporary zarr store at
+        ``~/.echopype/temp_output/parsed2zarr_temp_files`` if needed,
+        when set to "auto".
```

Comment:
Suggest the following wording changes. Also, from the description `destination_path="auto"` is the default, but in the code the default is `None`. Which one is it?

```suggestion
        The path to a swap directory when the memory footprint is large.
        This path can be a remote path like s3://bucket/swap_dir.
        When ``destination_path="auto"`` (default),
        a temporary zarr store at ``~/.echopype/temp_output/parsed2zarr_temp_files``
        will be created when necessary.
```